src/applications/vaos/utils/selectors.js

```diff
@@ -38,6 +38,26 @@ import {
   getVARFacilityId,
 } from '../services/appointment';
 
+export const vaosApplication = state => toggleValues(state).vaOnlineScheduling;
+export const vaosCancel = state => toggleValues(state).vaOnlineSchedulingCancel;
+export const vaosRequests = state =>
+  toggleValues(state).vaOnlineSchedulingRequests;
+export const vaosCommunityCare = state =>
+  toggleValues(state).vaOnlineSchedulingCommunityCare;
+export const vaosDirectScheduling = state =>
+  toggleValues(state).vaOnlineSchedulingDirect;
+export const vaosPastAppts = state =>
+  toggleValues(state).vaOnlineSchedulingPast;
+export const vaosVSPAppointmentNew = state =>
+  toggleValues(state).vaOnlineSchedulingVspAppointmentNew;
+export const vaosExpressCare = state =>
+  toggleValues(state).vaOnlineSchedulingExpressCare;
+export const vaosExpressCareNew = state =>
+  toggleValues(state).vaOnlineSchedulingExpressCareNew;
+export const vaosFlatFacilityPage = state =>
+  toggleValues(state).vaOnlineSchedulingFlatFacilityPage;
+export const selectFeatureToggleLoading = state => toggleValues(state).loading;
+
```

Comment:
Moved these feature toggle selectors up in the file so we can use them in other selectors.